### PR TITLE
rails-blog-app : Blog app views

### PR DESCRIPTION
**Pull request by micheaol**
## In this PR:

- Implement the design from the sneak peek wireframes.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,23 @@
...
+</section>
+
+<% if @user.bio.present? %>
+	<div class = 'user-bio'>
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+<% end %>
+<%= render partial: 'posts/post', collection: @post %> 
+<%= link_to "See all post", user_posts_path(@user.id), class: 'see-posts-button' %>
```
Comment (rukundoeric) [inline app/views/users/show.html.erb]: - [ ] <b>REQUIRED:</b> The project requires to render the recent posts of the user here. Kindly do so to comply with the project requirements.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @micheaol, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/users/show.html.erb (hunk 1 above)
Comment (micheaol) [reply app/views/users/show.html.erb]: @rukundoeric,

Thank you for the feedback.

This is exactly what I have done here. Please check the image below. What do I need to do next?:

image

image

Comment (micheaol) [reply app/views/users/show.html.erb]: @rukundoeric ,
Kindly reply to the message with the way forward. Thank you.

**Comment by DeliceLydia** [comment]
- [ ] Please fix this error you have in your migration `AddUserRefToPosts`
migr

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 

## Hi @micheaol, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**